examples/decoding/plot_receptive_field.py

```diff
@@ -0,0 +1,115 @@
+"""
+=========================================
+Receptive Field Estimation and Prediction
+=========================================
+
+This example re-creates figures from a paper from the Lalor group which
+demos the mTRF toolbox in Matlab [1]. We will show how the ReceptiveField
```

Comment:
Yeah `.. [1]` is the correct one, the indented is old and less good (check out how each renders to see why). Feel free to correct those here if you want.